### DISCUSSION TITLE
no longer available

### DISCUSSION THREAD

**Discussion by chankwon8** [huiyadanli/RevokeMsgPatcher#425] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112758158/188267606-c7fe2e99-c748-4490-b5bf-76b50f3967c7.png)
![image](https://user-images.githubusercontent.com/112758158/188267615-cd5287a8-42bc-46ee-9175-a2e4d6d6ad30.png)
no longer available

**Comment by huiyadanli** [comment, marked as answer]
手动选择微信目录

**Reply by chankwon8** [reply]
OK
